src/renderer/components/uielements/info/InfoIcon.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+
+import { InfoIcon } from './InfoIcon'
+
+storiesOf('Components/InfoIcon', module).add('default', () => <InfoIcon tooltip="testing info icon" />)
```

Comment:
Pls use syntax of latest `storybook` (some of our stories aren't updated - so you might need to check https://storybook.js.org/docs/react/writing-stories/introduction#how-to-write-stories)